### ISSUE TITLE
DuckDNS not generating fullchain.pem and privkey.pem files

### ISSUE THREAD

**Issue by jd2121** [home-assistant/hassio-addons#1492]
I followed [the official guide](https://www.home-assistant.io/blog/2017/09/27/effortless-encryption-with-lets-encrypt-and-duckdns/) to install DuckDNS on a fresh install of Home Assistant 4.11 32-bit on my Raspberry Pi 4 Model B 4GB. 

It does not appear to place (possibly not create) the fullchain.pem and privkey.pem files into the /SSL directory... so it does not enable SSL and causes issues if I try to input the http: section in the configuration.yaml file.

**DuckDNS Configuration:**
```
lets_encrypt:
  accept_terms: true
  certfile: /ssl/fullchain.pem
  keyfile: /ssl/privkey.pem
token: MY TOKEN
domains:
  - MYDOMAIN.duckdns.org
aliases: []
seconds: 300
```

**Configuration.yaml:**
```
http:
  base_url: MYDOMAIN.duckdns.org
  ssl_certificate: /ssl/fullchain.pem
  ssl_key: /ssl/privkey.pem
```

**DuckDNS Log:**
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
# INFO: Using main config file /data/workdir/config
+ Account already registered!
[16:29:47] INFO: OK
MYIP
NOCHANGE
# INFO: Using main config file /data/workdir/config
Processing MYDOMAIN.duckdns.org
 + Checking domain name(s) of existing cert... unchanged.
 + Checking expire date of existing cert...
 + Valid till Oct 19 15:43:43 2020 GMT Certificate will not expire
(Longer than 30 days). Skipping renew!
```
Does anyone have an idea of what I am doing wrong? I am new to Home Assistant and have spent hours trying different combinations of settings trying to get this to generate the certificates without success.

**Comment by ludeeus** [comment]
Use the settings from the example in the docs for the addon.
```
  certfile: fullchain.pem
  keyfile: privkey.pem
``` 
Then this in HA:
```
  ssl_certificate: /ssl/fullchain.pem
  ssl_key: /ssl/privkey.pem
```

**Comment by jd2121** [comment]
After I couldn't figure it out within Duck DNS; I installed Lets Encypt it generated the files... so after playing around with some addons and such for HA, I decided to start fresh with a clean install with everything I decided to implement. The first thing I did was install the Duck DNS addon and config it. It worked perfectly the first time. I am not sure what went wrong before, but the issue is fixed.
